### PR TITLE
Update apiview token prefix

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Update APIvIew token schema and APIView server to support hasPrefixSpace property at token level.

### CODE REVIEW

File: src/dotnet/APIView/APIView/Model/V2/ReviewLine.cs [hunk 1]
```diff
@@ -106,6 +106,7 @@ private string ToString(bool includeAllTokens)
             StringBuilder sb = new();
             foreach (var token in filterdTokens)
             {
+                sb.Append(token.HasPrefixSpace == true ? " " : string.Empty);
                 sb.Append(token.Value);
                 sb.Append(token.HasSuffixSpace == true ? " " : string.Empty);
```
Comment (tjprescott) [inline src/dotnet/APIView/APIView/Model/V2/ReviewLine.cs:111]: This logic needs to be smarter. If the previous token has "HasSuffixSpace" set to true, then this needs to be ignored so you don't end up with a double space.

File: src/dotnet/APIView/APIViewWeb/Helpers/CodeFileHelpers.cs [hunk 2]
```diff
@@ -253,6 +253,12 @@ private static CodePanelRowData GetCodePanelRowData(CodePanelData codePanelData,
             // Convert ReviewToken to UI required StructuredToken
             foreach (var token in reviewLine.Tokens)
             {
+                if (token.HasPrefixSpace == true)
+                {
+                    var spaceToken = StructuredToken.CreateSpaceToken();
+                    spaceToken.Value = " ";
+                    tokensInRow.Add(spaceToken);
+                }
```
Comment (tjprescott) [inline src/dotnet/APIView/APIViewWeb/Helpers/CodeFileHelpers.cs:261]: Same comment here about avoiding a double-space.

**Review by tjprescott — CHANGES_REQUESTED** [review]

File: src/dotnet/APIView/APIView/Model/V2/ReviewLine.cs (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIView/Model/V2/ReviewLine.cs]: Done.

**Review by tjprescott — APPROVED** [review]
Thank you so much @praveenkuttappan!

**Comment by jeremymeng** [comment]
This would help simplify current JS api parser code!

**Pull request merged**